.github/workflows/realm-dart-linux.yml

```diff
@@ -37,14 +37,10 @@ jobs:
       - name: Build Realm Dart for Linux
         run: ./scripts/build-linux.sh
 
-      # Note: This workflow uses the latest stable version of the Dart SDK.
-      # You can specify other versions if desired, see documentation here:
-      # https://github.com/dart-lang/setup-dart/blob/main/README.md
-      # - uses: dart-lang/setup-dart@v1
       - name : Setup Dart SDK
         uses: dart-lang/setup-dart@main
         with:
-          sdk: beta
+          sdk: stable
```

Comment:
`realm-dart-linux.yaml` and `realm-generator.yaml` already contain the changes, so we should not change it here. Could you please remove these files from the PR

Reply:
this one is in master now. ignore it here

Reply:
Done. Merged from `master.`